src/test/java/org/springframework/data/elasticsearch/core/CriteriaQueryMappingUnitTests.java

```diff
@@ -109,6 +108,158 @@ void shouldMapNamesAndConvertValuesInCriteriaQuery() throws JSONException {
 		assertEquals(expected, queryString, false);
 	}
 
+	@Test
+	// DATAES-1668
```

Comment:
```suggestion
	// #1668
```
DATAES was when we were using Jira, now we just use the #NNN